### ISSUE TITLE
🐛 Not Applicable flutter 3.7 

### ISSUE THREAD

**Issue by whaleHan** [bdlukaa/fluent_ui#697]
Error (Xcode): ../../../.pub-cache/hosted/pub.flutter-io.cn/fluent_ui-4.1.5/lib/src/controls/inputs/slider.dart:359:8: Error: The method '_CustomTrackShape.paint' has fewer named arguments than those of overridden method 'RoundedRectSliderTrackShape.paint'.

Could not build the application for the simulator.
Error launching application on iPhone 14 Pro Max.

**Comment by cranst0n** [comment]
Until a new release is published you can get around it by editing your `pubspec.yaml`:

```
  fluent_ui:              #^4.1.4
    git:
      url: https://github.com/bdlukaa/fluent_ui.git
      ref: master
```

**Comment by bdlukaa** [comment]
Published version 4.2.0, which should fix this!
